mta/application.properties

```diff
@@ -1,7 +1,11 @@
+# MTA
 mta.url = ${MTA_URL:http://mta-ui.openshift-mta.svc.cluster.local:8080}
 quarkus.rest-client.mta_json.url = ${mta.url}/hub
-quarkus.rest-client.notifications.url=${BACKSTAGE_NOTIFICATIONS_URL:http://backstage-backstage.rhdh-operator/api/notifications/}
 #quarkus.rest-client.mta_json.auth.basicAuth.username = username
 #quarkus.rest-client.mta_json.auth.basicAuth.password = password
 
+# Backstage Notifications service
+quarkus.rest-client.notifications.url=${BACKSTAGE_NOTIFICATIONS_URL:http://backstage-backstage.rhdh-operator}
+quarkus.openapi-generator.notifications.auth.BearerToken.bearer-token=${NOTIFICATIONS_BEARER_TOKEN}
```

Comment:
Please set a default value, otherwise you will have an error in the e2e:
```
2024-07-10T12:14:41.487270118Z stderr F Caused by: java.util.NoSuchElementException: SRCFG00011: Could not expand value NOTIFICATIONS_BEARER_TOKEN in property quarkus.openapi-generator.notifications.auth.BearerToken.bearer-token
```

`quarkus.openapi-generator.notifications.auth.BearerToken.bearer-token=${NOTIFICATIONS_BEARER_TOKEN:-}`